test/e2e/seeder/anvil-seeder.js

```diff
@@ -44,42 +50,56 @@ class AnvilSeeder {
     });
 
     if (contractName === SMART_CONTRACTS.NFTS) {
-      const transaction = await walletClient.sendTransaction({
-        from: fromAddress,
-        data: contractConfig.abi.encodeFunctionData('mintNFTs', [1]),
-        to: receipt.contractAddress,
-      });
-      await publicClient.getTransactionReceipt({ hash: transaction.hash });
+      const mintOptions = {
+        address: receipt.contractAddress,
+        abi: contractConfig.abi,
+        functionName: 'mintNFTs',
+        args: [1],
+        account: fromAddress,
```

Comment:
just a comment, it is strange that `account` and `address` are used instead of `from` and `to`. `account` and `address` are NOT properties of a transaction, and in practice mean the same thing.

I'm so flabbergasted by this I had to ask the devs: https://github.com/wevm/viem/discussions/3465

Reply:
yeah I also found it a bit counter-intuitive :/ thanks for following up in their repo :heart_eyes: